examples/example-node-telemetry/README.md

```diff
@@ -0,0 +1,111 @@
+# Telemetry demo app using data ingest and Atlas Charts
+
+A [node.js](https://nodejs.org) application to demonstrate how to use [Atlas Device Sync](https://www.mongodb.com/atlas/app-services/device-sync) and [Realm JavaScript](https://github.com/realm/realm-js) to read sensor data, store the data in [Atlas](https://www.mongodb.com/atlas), and visualize it with [Charts](https://www.mongodb.com/products/charts).
+
+The file structure is as follows:
+
+```
+├── README.md
+├── package.json
+├── src
+│   ├── app.ts
+│   ├── config.ts
+│   └── models
+│       ├── machine_info.ts
+│       └── sensor_reading.ts
+```
+
+* `src/app.ts` - the actual application
+* `src/config.ts` - contains the configuration (app id)
+* `src/models/` - the model classes
+
+## Altas App Services
+
+To use the app together with Atlas App Services, you need to create an Atlas database and collection. Moreover, you need to create App service and enable Device Sync. 
```

Comment:
Making some updates to align with [this README](https://github.com/realm/realm-js/blob/main/examples/node-connection-and-error/README.md).

```suggestion
## Getting Started

### Prerequisites

* [Node.js](https://nodejs.org/)

### Set Up an Atlas App Services App

To sync Realm data you must first:

1. [Create an App Services App](https://www.mongodb.com/docs/atlas/app-services/manage-apps/create/create-with-ui/)
2. [Enable Anonymous Authentication](https://www.mongodb.com/docs/atlas/app-services/authentication/anonymous/)
3. [Enable Flexible Sync](https://www.mongodb.com/docs/atlas/app-services/sync/configure/enable-sync/) with **Development Mode** on.

After running the client and seeing the available collection in Atlas, [set write permissions](https://www.mongodb.com/docs/atlas/app-services/rules/roles/#define-roles---permissions) for the collection.
```